app/controllers/dashboard/trainer_controller.rb

```diff
@@ -42,6 +29,25 @@ def review_card
 
   private
 
+  def first_repeating_or_pending_card
+    if current_user.current_block
+      current_user.current_block.cards.first_repeating_or_pending_card
+    else
+      current_user.cards.first_repeating_or_pending_card
+    end
+  end
+
+  def handle_correct_answer(distance)
+    if distance == 0
+      flash[:notice] = t(:correct_translation_notice)
+    else
+      flash[:alert] = t 'translation_from_misprint_alert',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.